### ISSUE TITLE
Chat moderation system

### ISSUE THREAD

**Issue by Fyrsta** [TranxCraft/iTranxCraft#34]
An automated bot to keep profanity and similar content out of the chat.

**Comment by mgejay** [comment]
Configurable in config for filtered words?

**Comment by Fyrsta** [comment]
@TheRevalation Yeah, most likely there will be a table in the database with a list of banned words.
